exonum/src/storage/keys.rs

```diff
@@ -276,3 +226,69 @@ impl StorageKey for String {
         unsafe { ::std::str::from_utf8_unchecked(buffer).to_string() }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    // Number of samples for fuzz testing
+    const FUZZ_SAMPLES: usize = 10_000;
+
+    macro_rules! test_storage_key_for_int_type {
+        (full $type:ident, $size:expr => $test_name:ident) => {
+            #[test]
+            fn $test_name() {
+                // Roundtrip
+                let mut buffer = [0u8; $size];
+                for x in $type::min_value()..$type::max_value() {
```

Comment:
This misses the `max_value` value itself. I don't know a beautiful way to write it correctly, only a kludge of `.chain(once($type::max_value()))`